### PR TITLE
Feature/app wrappers

### PR DESCRIPTION
**Pull request by StyleT**

### CODE REVIEW

File: ilc/server/registry/Registry.js [hunk 1]
```diff
@@ -52,6 +52,18 @@ module.exports = class Registry {
             responseType: 'json',
         });
 
+        //FIXME: remove TEMP code:
```
Comment (nightnei) [inline ilc/server/registry/Registry.js:55]: can we remove it now?

File: ilc/server/tailor/configs-injector.spec.js [hunk 2]
```diff
@@ -201,7 +203,7 @@ describe('configs injector', () => {
         };
 
         const getSpaConfig = () => {
-            const pickApp = _.pick(['spaBundle', 'cssBundle', 'dependencies', 'props', 'kind']);
+            const pickApp = _fp.pick(['spaBundle', 'cssBundle', 'dependencies', 'props', 'kind', 'wrappedWith']);
```
Comment (nightnei) [inline ilc/server/tailor/configs-injector.spec.js:206]: do we really need here `lodash/fp`?

File: ilc/server/tailor/server-router.js [hunk 3]
```diff
@@ -8,95 +7,130 @@ const {makeAppId} = require('../../common/utils');
...
-                        `More then one primary slot "${slotName}" found for "${reqProps.reqUrl}".\n` +
+                        `More then one primary slot "${row.name}" found for "${this.#url}".\n` +
                         'Make it regular to avoid unexpected behaviour.'
                     );
                 }
-                delete ssrOpts.primary;
             }
 
-            ssrOpts.src = url.toString();
+            ssrOpts.appProps = _.merge({}, appInfo.props, row.props);
```
Comment (nightnei) [inline ilc/server/tailor/server-router.js:71]: It's not critical, but I propose to use everywhere `deepmerge` in the future, to not have inconsistency results and unpredictable behavior

for example https://github.com/namecheap/ilc/pull/247/files#diff-8d15e4857d951cee8de1012b2eab00c6d624677f8fd076209397dcf99710a7fdR56

File: ilc/tests/helpers.js [hunk 4]
```diff
@@ -1,9 +1,9 @@
-const _ = require('lodash');
+const deepmerge = require('deepmerge');
```
Comment (nightnei) [inline ilc/tests/helpers.js:1]: 👍

**Review by nightnei — APPROVED** [review]

File: ilc/server/registry/Registry.js (hunk 1 above)
Comment (StyleT) [reply ilc/server/registry/Registry.js:55]: see https://github.com/namecheap/ilc/pull/249/files#diff-569e299812f8148a25f8ef50aaa9f5cab818b6395ffcbb9310658a3f4b49074e

File: ilc/server/tailor/configs-injector.spec.js (hunk 2 above)
Comment (StyleT) [reply ilc/server/tailor/configs-injector.spec.js:206]: It's convenient here... Easier to keep, IMO

File: ilc/tests/helpers.js (hunk 4 above)
Comment (StyleT) [reply ilc/tests/helpers.js:1]: ))

File: ilc/server/tailor/server-router.js (hunk 3 above)
Comment (StyleT) [reply ilc/server/tailor/server-router.js]: fixed

**Review by oleh-momot — APPROVED** [review]

**Pull request merged**